textattack/constraints/grammaticality/language_models/google_language_model/google_language_model.py

```diff
@@ -34,24 +34,18 @@ def __init__(self, top_n=None, top_n_per_index=None, print_step=False):
     def check_compatibility(self, transformation):
         return isinstance(transformation, WordSwap)
 
-    def _check_constraint_many(self, x, x_adv_list, original_text=None):
+    def _check_constraint_many(self, transformed_texts, current_text, original_text=None):
         """
-        Returns the `top_n` of x_adv_list, as evaluated by the language 
+        Returns the `top_n` of transformed_texts, as evaluated by the language 
         model. 
-
-        Args:
-            x:
-            x_adv_list:
-            original_text (:obj:`type`, optional): Defaults to None. 
-
         """
         # @TODO Allow user to implement perplexity threshold
```

Comment:
mind removing this as well? this is no longer something we want to do, since we did it in the LanguageModel class, and we don't really want to encourage people to use this class anyway :)